freqtrade/freqtradebot.py

```diff
@@ -121,6 +121,8 @@ def update():
                     self._schedule.every().day.at(t).do(update)
         self.last_process = datetime(1970, 1, 1, tzinfo=timezone.utc)
 
+        self.strategy.bot_start()
```

Comment:
I can't think of why this would need to be in edge, but maybe it should be above `# Initializing Edge only if enabled`?

Reply:
it's not about edge itself (maybe i wasn't 100% clear) - but about the assumption that users can rely on this to be ran when the strategy initializes.

Imagine a usecase where you load your machine learning model in this function (we assume this to be the `__init__`  for users - no? (otherwise it'd be kinda pointless).

if you run edge (edge-cli, to be precise) - then you'd also want to run this - otherwise your populate_* methods would fail - as your model would not be loaded.

we'd probably also want this for plotting ... but it's no problem to omit it in some areas if we have that documented (like we do for bot-loop_start - which also ain't called in the plotting scenario.